pandas/tests/plotting/test_frame.py

```diff
@@ -367,6 +367,61 @@ def test_subplots(self):
             for ax in axes:
                 assert ax.get_legend() is None
 
+    def test_groupby_boxplot_sharey(self):
+        # https://github.com/pandas-dev/pandas/issues/20968
+        # sharey can now be switched check whether the right
+        # pair of axes is turned on or off
+
+        def _assert_ytickslabels_visibility(axes, expected):
```

Comment:
do we have similar functions at a top level? maybe we should

Reply:
Sorry, I am new to pandas. I am not sure whether I understand what you mean. Are you saying, I should move the function somewhere else? If yes, where exactly should the function go?

Reply:
you can make this a function on the class itself rather than defininig it inside the function